bip-undef-1.mediawiki

```diff
@@ -0,0 +1,170 @@
+<pre>
+  BIP: ???
+  Title: Peer-to-Peer Communication Encryption
+  Author: Jonas Schnelli <dev@jonasschnelli.ch>
+  Status: Draft
+  Type: Standards Track
+  Created: 2016-03-23
+</pre>
+
+== Abstract ==
+
+This BIP describes an alternative way that a peer can encrypt their communication between a selective subset of remote peers.
+
+== Motivation ==
+
+The current p2p communication is unencrypted and can be analyzed and modified by many network authority (ISP, WiFi providers, router-firmware, etc.). Mostly this is negligible because of the nature of Bitcoins trust model, however for SPV nodes this can have significant privacy impacts [1] and could reduce the censorship-resistance of a peer.
```

Comment:
Suggestion for first sentence:
The Bitcoin network does not encrypt communication between peers today.  This opens up security issues (eg: traffic manipulation by others) and allows for mass surveillance / analysis of bitcoin users.